airflow/providers_manager.py

```diff
@@ -175,6 +175,9 @@ def _create_customized_form_field_behaviours_schema_validator():
 
 
 def _check_builtin_provider_prefix(provider_package: str, class_name: str) -> bool:
+    if "bundles" in provider_package:
+        # TODO: remove this when this package is moved to providers directory
```

Comment:
```suggestion
        # TODO: AIP-66: remove this when this package is moved to providers directory
```

Let's leave this breadcrumb for our future selves